src/context.rs

```diff
@@ -279,6 +279,7 @@ impl<'a> Context<'a> {
             "fish" => Shell::Fish,
             "ion" => Shell::Ion,
             "powershell" => Shell::PowerShell,
+            "pwsh" => Shell::PowerShell,
```

Comment:
To make it more obvious that they map to the same value
```suggestion
            "powershell" | "pwsh" => Shell::PowerShell,
```